topology/University of Michigan/AGLT2/AGLT2.yaml

```diff
@@ -237,6 +237,94 @@ Resources:
       KSI2KMin: 0
       StorageCapacityMax: 6879
       StorageCapacityMin: 6879
+  AGLT2_XCache:
+    Active: true
+    ContactLists:
+      Administrative Contact:
+        Primary:
+          ID: 737e25f3cb6a16c6c1846ee2601b5e5b102bc157
+          Name: Wenjing Wu
+        Secondary:
+          ID: fe2ecaf4ad4a0923d1be605ad55a8ca71f67eb66
+          Name: Philippe Laurens
+        Tertiary:
+          ID: a833f94ad3840fe779702409c6b778d79aaed76f
+          Name: Shawn McKee
+      Security Contact:
+        Primary:
+          ID: 737e25f3cb6a16c6c1846ee2601b5e5b102bc157
+          Name: Wenjing Wu
+        Secondary:
+          ID: fe2ecaf4ad4a0923d1be605ad55a8ca71f67eb66
+          Name: Philippe Laurens
+        Tertiary:
+          ID: a833f94ad3840fe779702409c6b778d79aaed76f
+          Name: Shawn McKee
+    Description: XCache service for AGLT2
+    FQDN: xcache.aglt2.org
+    ID: 1169
+    Services:
+      XRootD cache server:
+        Description: XCache service
+        Details:
+          hidden: false
+    VOOwnership:
+      ATLAS: 90
+    WLCGInformation:
+      APELNormalFactor: 0
+      AccountingName: US-AGLT2
+      HEPSPEC: 0
+      InteropAccounting: true
+      InteropBDII: true
+      InteropMonitoring: true
+      KSI2KMax: 0
+      KSI2KMin: 0
+      StorageCapacityMax: 6879
+      StorageCapacityMin: 6879
```

Comment:
```suggestion
```

@ivukotic said that he's actually going to remove the corresponding CRIC registrations because 1) they're not quite right and 2) the XCaches are directly registered in VPS